src/services/cronJobs/syncUsersModelScore.ts

```diff
@@ -0,0 +1,71 @@
+import { schedule } from 'node-cron';
+import { spawn, Worker, Thread } from 'threads';
+import config from '../../config';
+import { logger } from '../../utils/logger';
+import {
+  findActiveQfRound,
+  findUsersWithoutMBDScoreInActiveAround,
+} from '../../repositories/qfRoundRepository';
+import { findUserById } from '../../repositories/userRepository';
+import { UserQfRoundModelScore } from '../../entities/userQfRoundModelScore';
+import { UserMBDScoreSyncWorker } from '../../workers/userMBDScoreSyncWorker';
+
+const cronJobTime =
+  (config.get('MAKE_UNREVIEWED_PROJECT_LISTED_CRONJOB_EXPRESSION') as string) ||
+  '0 0 * * * *';
+
+const qfRoundUsersMissedMBDScore = Number(
+  process.env.QF_ROUND_USERS_MISSED_SCORE || 0,
+);
+
+const workerOptions = {
+  concurrency: Number(
+    process.env.USER_SCORE_SYNC_THREADS_POOL_CONCURRENCY || 1,
+  ),
+  name:
+    process.env.USER_SCORE_SYNC_THREADS_POOL_NAME || 'ProjectFiltersThreadPool',
+  size: Number(process.env.USER_SCORE_SYNC_THREADS_POOL_SIZE || 4),
+};
+
+export const runCheckPendingUserModelScoreCronjob = () => {
+  logger.debug(
+    'runCheckPendingUserModelScoreCronjob() has been called, cronJobTime',
+    cronJobTime,
+  );
+  schedule(cronJobTime, async () => {
+    await updateUsersWithoutMBDScoreInRound();
+  });
+};
+
+export const updateUsersWithoutMBDScoreInRound = async () => {
+  const worker = await spawn(
+    new Worker('../../workers/userMBDScoreSyncWorker'),
+  );
+  const userIds = await findUsersWithoutMBDScoreInActiveAround();
+  const activeQfRoundId =
+    (await findActiveQfRound())?.id || qfRoundUsersMissedMBDScore;
+  if (!activeQfRoundId || activeQfRoundId === 0) return;
+
+  if (userIds.length === 0) return;
+
+  for (const userId of userIds) {
+    try {
+      const userWallet = await findUserById(userId);
```

Comment:
@CarlosQ96 I think findUserById() returns a user not a wallet address, so you are passing user instead of userWallet, am I right?

Reply:
You are right I have a false positive here because of the Mock Adapter that returns 1 instead of hitting a service. Ill fix it. (wanted to avoid hitting a http request because it can take anywhere from 3 to 20 seconds).

Reply:
pushed hotfix

Reply:
Cool!